e2e/tests/vscode.test.ts

```diff
@@ -5,14 +5,17 @@ import { cleanupRepo, uninstallExtension } from '../utilities/utils';
 // TODO : Get repo URL from fixtures
 const repoUrl = 'https://github.com/konveyor-ecosystem/coolstore';
 
+
+test.describe.configure({ mode: 'serial' });
 test.describe('VSCode Tests', () => {
   let vscodeApp: VSCode;
 
   test.beforeAll(async () => {
-    test.setTimeout(60000);
-    const executablePath =
-      process.env.VSCODE_EXECUTABLE_PATH || '/usr/share/code/code';
-    vscodeApp = await VSCode.init(executablePath, repoUrl, 'coolstore');
+    console.log(`[beforeAll] Called at: ${new Date().toISOString()}`);
+    test.setTimeout(120000);
+      const executablePath =
+        process.env.VSCODE_EXECUTABLE_PATH || '/usr/share/code/code';
```

Comment:
We need to modify code to use 
https://github.com/konveyor/kai-ci/blob/main/.env.example#L16  for windows .